tests/snappi_tests/multidut/bgp/files/bgp_outbound_helper.py

```diff
@@ -0,0 +1,632 @@
+                       duthosts[2].hostname))
+    logger.info('\n')
+    logger.info('T1 Dut AS Number: {}'.format(T1_DUT_AS_NUM))
+    logger.info('T2 Dut AS Number: {}'.format(T2_DUT_AS_NUM))
+    t1_bgp = (
+        "vtysh "
+        "-c 'configure terminal' "
+        "-c 'router bgp %s' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv4 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv6 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s soft-reconfiguration inbound' "
+        "-c 'neighbor %s route-map RM_SET_SRC6 in' "
+        "-c 'maximum-paths 64' "
+        "-c 'exit' "
+    )
+    t1_bgp %= (
+        T1_DUT_AS_NUM, t1_t2_snappi_ipv4_list[index], T2_DUT_AS_NUM, t1_t2_snappi_ipv4_list[index],
+        t1_t2_snappi_ipv6_list[index], T2_DUT_AS_NUM, t1_t2_snappi_ipv6_list[index],
+        t1_t2_snappi_ipv6_list[index], t1_t2_snappi_ipv6_list[index]
+    )
+    duthosts[0].shell(t1_bgp)
+    logger.info('Configuring BGPv4 and BGP+ Neighbor {} {} in {}'.
+                format(t1_t2_snappi_ipv4_list[index], t1_t2_snappi_ipv6_list[index], duthosts[0].hostname))
+
+    t2_downlink_bgp = (
+        "vtysh -n %s "
+        "-c 'configure terminal' "
+        "-c 'route-map RM_SET_SRC6 permit 10' "
+        "-c 'on-match next' "
+        "-c 'set ipv6 next-hop prefer-global' "
+        "-c 'exit' "
+        "-c 'ip nht resolve-via-default' "
+        "-c 'ipv6 nht resolve-via-default' "
+        "-c 'ipv6 protocol bgp route-map RM_SET_SRC6' "
+        "-c 'router bgp %s' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv4 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv6 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s soft-reconfiguration inbound' "
+        "-c 'neighbor %s route-map RM_SET_SRC6 in' "
+        "-c 'maximum-paths 64' "
+        "-c 'exit' "
+    )
+    t2_downlink_bgp %= (
+        t2_side_interconnected_port['asic_value'][-1], T2_DUT_AS_NUM,
+        t1_t2_dut_ipv4_list[index], T1_DUT_AS_NUM, t1_t2_dut_ipv4_list[index],
+        t1_t2_dut_ipv6_list[index], T1_DUT_AS_NUM, t1_t2_dut_ipv6_list[index],
+        t1_t2_dut_ipv6_list[index], t1_t2_dut_ipv6_list[index]
+    )
+    duthosts[2].shell(t2_downlink_bgp)
+    logger.info('Configuring BGPv4 and BGP+ Neighbor {} {} in {} frr {}'.format(t1_t2_dut_ipv4_list[index],
+                t1_t2_dut_ipv6_list[index], duthosts[2].hostname, t2_side_interconnected_port['asic_value'][-1]))
+
+    logger.info('--------------- T2 Uplink - Tgen Section --------------------')
+
+    index = 0
+    for asic_value, portchannel_info in t2_uplink_portchannel_members[duthosts[1].hostname].items():
+        for portchannel, ports in portchannel_info.items():
+            duthosts[1].command('sudo config portchannel -n {} add {} \n'.
```

Comment:
Can we using this function instead of calling commands directly here. This function handles single and multi asic devices https://github.com/sonic-net/sonic-mgmt/blob/0e5793c3e360c723dc1447a7c7cbe748098ec3e1/tests/common/devices/sonic_asic.py#L519

Reply:
will address this a separate PR in phase2